MANIFEST.in

```
@@ -8,9 +8,54 @@ include yt/utilities/mesh_types.yaml
 exclude yt/utilities/lib/cykdtree/c_kdtree.cpp
 prune tests
 prune answer-store
-recursive-include yt *.py *.pyx *.pxi *.pxd *.h *.hpp README* *.txt LICENSE* *.cu
+recursive-include yt *.py *.pyx *.pxi *.pxd  README* *.txt LICENSE* *.cu
 recursive-include doc *.rst *.txt *.py *.ipynb *.png *.jpg *.css *.html
 recursive-include doc *.h *.c *.sh *.svgz *.pdf *.svg *.pyx
```

Now that we are at it, there are no `.svgz` nor `.pdf` in the doc folder, unless they are generated on our CI?
```suggestion
recursive-include doc *.h *.c *.sh *.svg *.pyx
```

---

I think the latex cheatsheet is a generated pdf. I don't know of generated `.svgz` files but I'd err on the side of caution and label this as out of scope here.

---

Fine, this honestly doesn't matter much and is not worth debating as far as I'm concerned :)